app/forms/lead_providers/report_schools_form.rb

```diff
@@ -17,13 +17,20 @@ class ReportSchoolsForm
     def save!
       ActiveRecord::Base.transaction do
         school_ids.each do |school_id|
-          partnership = Partnership.create!(
+          partnership = Partnership.find_or_initialize_by(
```

Comment:
We're missing lots of the functionality from confirm_schools_form here. What about schools without a SchoolCohort?

Reply:
Ugh, I think the problem here is much bigger - `confirm_schools_form` is not used any more. It seems we (most likely me) missed quite a few silent merge conflicts...